docker/solr/8/debian10/8.6/scripts/docker-entrypoint.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Enable bash debug if DEBUG_DOCKER_ENTRYPOINT exists.
+if [[ "${DEBUG_DOCKER_ENTRYPOINT}" = "true" ]]; then
```

Comment:
What do you think about the suggestion below? 

```
if [[ "${DEBUG_DOCKER_ENTRYPOINT}" = "true" ]]; then
    VERBOSE="yes"
fi
```

This way, you can follow the pattern we use in other containers and even keep the original behaviour without changing the original the code below.

Reply:
IMHO, if there is already VERBOSE option, no need to change script